transaction/builder/src/transaction_builder.rs

```diff
@@ -3247,29 +3086,14 @@ pub mod transaction_builder_tests {
             let mut memo_builder = BurnRedemptionMemoBuilder::new([2u8; 64]);
             memo_builder.enable_destination_memo();
 
-            let mut transaction_builder = TransactionBuilder::new(
-                block_version,
-                Amount::new(10, token_id),
-                fog_resolver.clone(),
-            )
-            .unwrap();
-
+            let mut transaction_builder = transaction_builder.clone();
             transaction_builder.set_fee(3).unwrap();
```

Comment:
this change threw me for a loop
I couldn't understand how the test used to pass
I saw the 10 fee being set up in the `new()` and overlooked the fee being reset to 3, not sure why  it was done that way instead of initialized to 3, but test seems a bit easier to follow now

Reply:
I think some tests used different amounts/fees/tokens just to exercise that functionality and not have everything use the exact same inputs/configuration.